std_msgs/Empty request
---
bool wallfound